src/ToolingSharedLibrary/ToolingExecutable/Edl/Parser.cpp

```diff
@@ -428,16 +430,21 @@ namespace EdlProcessor
     void EdlParser::ParseFunctions(const FunctionKind& function_kind)
     {
         ThrowIfExpectedTokenNotNext(LEFT_CURLY_BRACKET);
-        auto& map = (function_kind == FunctionKind::Trusted)
-            ? m_trusted_functions
-            : m_untrusted_functions;
+        std::reference_wrapper<std::unordered_map<std::string, Function>> func_map = m_trusted_functions_map;
```

Comment:
>std::reference_wrapper [](http://example.com/codeflow?start=8&length=22)

TIL!

(personally I would have done lambda that returns the value, so you can just store it in a const auto&)